repairnator/repairnator-core/src/test/java/fr/inria/spirals/repairnator/TestUtils.java

```diff
@@ -0,0 +1,31 @@
+package fr.inria.spirals.repairnator;
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class TestUtils {
+
+    @Test
+    public void testMatchesGithubRepoUrlWithRightRepoUrl() {
+        String repoUrl = "https://github.com/Spirals-Team/repairnator";
+
+        assertThat(Utils.matchesGithubRepoUrl(repoUrl), is(true));
+    }
+
+    @Test
+    public void testMatchesGithubRepoUrlWithWrongRepoUrl() {
+        String repoUrl = "https://github.com/Spirals-Team/repairnator/";
+
+        assertThat(Utils.matchesGithubRepoUrl(repoUrl), is(false));
+    }
+
+    @Test
+    public void testMatchesGithubRepoUrlWithWrongRepoUrl2() {
```

Comment:
Could you add a test with project containing numbers?

Reply:
Sure. It's done, and it works for numbers.

However, I just realized that repo name can contain dot, and also I checked the rule for user name ("Username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen and is already taken" from GitHub). I will do some changes for these things. I tell you when it's done.